Core/Aspects/Caching/CacheAspect.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Castle.DynamicProxy;
+using WebNetSample.Core.CrossCuttingConcerns.Caching;
+using WebNetSample.Core.Utilities.Interceptors;
+using WebNetSample.Core.Utilities.IoC;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace WebNetSample.Core.Aspects.Caching;
+
+public class CacheAspect : MethodInterception
+{
+    private readonly ICacheService _cacheService;
+    public readonly int duration;
```

Comment:
You can use underscore prefix here as well to make it consistent with your `_cacheService`

Reply:
I did it with this. because of this usage in ImageCachingMiddleware actually.

`MaxAge = TimeSpan.FromMinutes(_cacheAspect.duration),`

Otherwise;

`MaxAge = TimeSpan.FromMinutes(_cacheAspect._duration),`

But I fixed as you wanted